lib/explorer/series.ex

```diff
@@ -9,6 +9,7 @@ defmodule Explorer.Series do
     * `:boolean` - Boolean
     * `:string` - UTF-8 encoded binary
     * `:binary` - Binary
+    * `:categorical` - UTF-8 encoded binary, but as categories.
```

Comment:
```suggestion
    * `:categorical` - UTF-8 encoded binary, but as categories
```

:D

Maybe let's sort them alphabetically too?